packages/components/addon/components/hds/text/index.hbs

```diff
@@ -2,7 +2,9 @@
   Copyright (c) HashiCorp, Inc.
   SPDX-License-Identifier: MPL-2.0
 }}
-
-{{#let (element this.componentTag) as |Tag|}}
-  <Tag class={{this.classNames}} {{style color=this.customColor}} ...attributes>{{yield}}</Tag>
-{{/let}}
\ No newline at end of file
+{{! IMPORTANT: we removed any extra newlines before/after the `let` to reduce the issues with unexpected whitespaces (see https://github.com/hashicorp/design-system/pull/1652) }}
+{{#let (element this.componentTag) as |Tag|}}<Tag
+    class={{this.classNames}}
+    {{style color=this.customColor}}
+    ...attributes
+  >{{yield}}</Tag>{{/let}}
```

Comment:
Would it work to use the "~" character instead?

Example: https://github.com/hashicorp/design-system/blob/main/packages/components/addon/components/hds/interactive/index.hbs

Reply:
No, it's a different thing (I tried myself what you suggested, but didn't work; unfortunately it's an exercise of trial and error for me, because I don't understand the logic of this; I suspect it's a matter of newlines, more than whitespaces)

Reply:
Ok thanks for explaining